crates/ruma-api/tests/manual_endpoint_impl.rs

```diff
@@ -28,8 +28,7 @@ const METADATA: Metadata = Metadata {
     description: "Add an alias to a room.",
     method: Method::PUT,
     name: "create_alias",
-    path: "/_matrix/client/r0/directory/room/:room_alias",
-    unstable_path: None,
+    unstable_path: Some("/_matrix/client/r0/directory/room/:room_alias"),
```

Comment:
This should be `r0_path` instead of unstable, no?

Reply:
This is an artifact from me copying things around (previously this was just `path:`), i'll properly fill out this one and the other ones.